app/assets/javascripts/components/nav/nav.jsx

```diff
@@ -136,6 +142,24 @@ const Nav = createReactClass({
           <CustomLink to="/admin" name="Admin" />
         </li>
       );
+      notifications = (
```

Comment:
Maybe this should only show for admins as well, unless/until there is the possibility of notifications for other users as well.